ccm_web/client/src/utils/enrollmentValidators.ts

```diff
@@ -66,3 +74,34 @@ export class LoginIDRowsValidator implements EnrollmentRowsValidator {
     return loginIDValidator.validate(loginIDs)
   }
 }
+
+export class SectionIdRowsValidator implements EnrollmentRowsValidator {
+  existingSectionIDs: number[]
+  constructor (existingSectionIDs: number[]) {
+    this.existingSectionIDs = existingSectionIDs
+  }
+
+  validate (sectionIds: string[]): EnrollmentInvalidation[] {
+    const invalidations: EnrollmentInvalidation[] = []
+    sectionIds.forEach((sectionId, i) => {
+      const result = validateNumberString(sectionId, sectionIdSchema)
+      if (result.transformedValue === undefined || !result.isValid) {
+        invalidations.push({
+          rowNumber: getRowNumber(i),
+          message: result.messages.length > 0 ? result.messages[0] : 'Value for the section ID was invalid.',
```

Comment:
Can this be worded like `Value for the section ID is invalid.`?

Reply:
Sure, done. https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/301/commits/db94057bb6ddf0a8b638cb992275fde9d709d4f7 I also removed "the" since the other default messages don't have that.